doc/man/man3/crypto_x25519_inverse.3monocypher

```diff
@@ -0,0 +1,102 @@
+.\" modification, are permitted provided that the following conditions are
+.\" met:
+.\"
+.\" 1. Redistributions of source code must retain the above copyright
+.\"    notice, this list of conditions and the following disclaimer.
+.\"
+.\" 2. Redistributions in binary form must reproduce the above copyright
+.\"    notice, this list of conditions and the following disclaimer in the
+.\"    documentation and/or other materials provided with the
+.\"    distribution.
+.\"
+.\" THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+.\" "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+.\" LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+.\" A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+.\" HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+.\" SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+.\" LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+.\" DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+.\" THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+.\" (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+.\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+.\"
+.\" ----------------------------------------------------------------------------
+.\"
+.\" Written in 2020 by Fabio Scotoni
+.\"
+.\" To the extent possible under law, the author(s) have dedicated all copyright
+.\" and related neighboring rights to this software to the public domain
+.\" worldwide.  This software is distributed without any warranty.
+.\"
+.\" You should have received a copy of the CC0 Public Domain Dedication along
+.\" with this software.  If not, see
+.\" <https://creativecommons.org/publicdomain/zero/1.0/>
+.\"
+.Dd March 24, 2020
+.Dt CRYPTO_X25519_INVERSE 3MONOCYPHER
+.Os
+.Sh NAME
+.Nm crypto_x25519_inverse
+.Nd X25519 scalar multiplication with the multiplicative inverse of a scalar
+.Sh SYNOPSIS
+.In monocypher.h
+.Ft void
+.Fo crypto_x25519_inverse
+.Fa "uint8_t blind_salt[32]"
+.Fa "const uint8_t private_key[32]"
+.Fa "const uint8_t curve_point"
+.Fc
+.Sh DESCRIPTION
+The
+.Fn crypto_x25519_inverse
+function performs scalar multiplication of the multiplicative inverse
+of a scalar for X25519.
+.Sy This is a highly advanced, specialized feature ;
+unless you are implementing a protocol that requires this specifically,
+.Sy you likely have no reason to be using these functions
+and are probably looking for
+.Xr crypto_key_exchange 3monocypher
+or
+.Xr crypto_x25519 3monocypher
+instead.
+Expect elliptic curve jargon on this page.
+.Pp
+This function is used, for example, with exponential blinding in
+oblivious pseudo-random functions (OPRFs).
+The arguments are:
+.Bl -tag -width Ds
+.It Fa blind_salt
+The output point.
+.It Fa private_key
+The private key (scalar) to use.
+This value's cofactor is cleared first,
```

Comment:
Missed it the first time around: this value is *clamped*, according to the X25519 standard. That clears its cofactor all right, but it also affects the main factor. I would say:

    This value is clamped first,
    then inverted (modulo the curve order),
    then has its cofactor cleared,
    and finally is used for scalar multiplication.

(You may find a better way to say this, but you get the idea.)